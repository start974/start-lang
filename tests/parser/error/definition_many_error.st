def 2 := 3

def a :=
  34

def b := _

def c := 6bob

def x : N2 := 3

