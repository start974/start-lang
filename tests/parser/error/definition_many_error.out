Error[201]: tests/parser/error/definition_many_error.st:1:5
1 | def 2 := 3
        ^ Expected identifier

Error[204]: tests/parser/error/definition_many_error.st:6:10
6 | def b := _
             ^ Expected not allowed to use wildcard '_' as an expression

Error[201]: tests/parser/error/definition_many_error.st:8:10
8 | def c := 6bob
             ^ Expected expression
